### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by IrenaPopovych7**
- [DEMO LINK](https://IrenaPopovych7.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://IrenaPopovych7.github.io/layout_moyo-header/report/html_report/)

- [ x] Header height is set in 1 place (for the links)
- [ x]  Content is vertically centered (for any header height)
- [ x]  CSS is used to show all letters in Uppercase (don't type them in HTML)
- [ x]  Logo is an image wrapped with a link
- [ x]  CSS Variable is used for a blue color
- [ x]  Pseudo-element is used for a blue line below the active link
- [ x]  Code follows all the [Code Style Rules ❗️]

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img class="logo"
+         src="./images/logo.png"
+         alt="logo"
+         >
```
Comment (TarasHoliuk) [inline src/index.html:22]:
```suggestion
        <img
          class="logo"
          src="./images/logo.png"
          alt="logo"
        >
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+header>a{
```
Comment (TarasHoliuk) [inline src/style.css:22]: don't use combined selector and tag selector for specific styles
add class to the element you want to style and use class selector

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+
+.menu_item {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  text-transform: uppercase;
+  margin-right: 20px;
+}
+
+.menu_item a {
```
Comment (TarasHoliuk) [inline src/style.css:47]: The same:

don't use combined selector and tag selector for specific styles
add class to the element you want to style and use class selector

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,75 @@
...
+  position: relative;
+  text-decoration: none;
+  color: inherit;
+  line-height: 60px;
+  transition: color 0.3s ease;
+  padding: 23px 0;
+}
+
+.menu_item a:hover,
```
Comment (TarasHoliuk) [inline src/style.css:56]: The same:

don't use combined selector and tag selector for specific styles
add class to the element you want to style and use class selector

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
GJ :+1: 
A few comments to improve your code

Note that in PR description wrong links (to layout_stars):
image

**Comment by IrenaPopovych7** [comment]
Exchange combined selector and tag selector for specific classes

**Pull request closed without merging**